C/plugins/storage/sqlite/plugin.cpp

```diff
@@ -40,8 +41,37 @@ const char *default_config = QUOTE({
 			"default" : "5",
 			"displayName" : "Pool Size",
 			"order" : "1"
-			}
-		});
+		},
+		"nReadingsPerDb" : {
+			"description" : "Number of readings tables per  database",
+			"type" : "integer",
+			"default" : "15",
+			"displayName" : "N Readings per database",
```

Comment:
We need to keep display names short but meaningful. Either change "N" to be "No." or drop it completely "Readings per database".